doc/cascadia/profiles.schema.json

```diff
@@ -339,17 +339,17 @@
         },
         "showTerminalTitleInTitlebar": {
           "default": true,
-          "description": "When set to true, titlebar displays the title of the selected tab. When set to false, titlebar displays 'Windows Terminal'.",
+          "description": "When set to true, titlebar displays the title of the selected tab. When set to false, titlebar displays \"Windows Terminal\".",
           "type": "boolean"
         },
         "snapToGridOnResize": {
           "default": false,
-          "description": "When set to true, the window will snap to the nearest character boundary on resize. When false, the window will resize 'smoothly'",
+          "description": "When set to true, the window will snap to the nearest character boundary on resize. When false, the window will resize \"smoothly\"",
           "type": "boolean"
         },
         "tabWidthMode": {
           "default": "equal",
-          "description": "Sets the width of the tabs.",
+          "description": "Sets the width of the tabs. Possible values include:\n -\"equal\" sizes each tab has the same width\n -\"titleLength\" sizes each tab to the length of its title",
```

Comment:
The sentence "equal sizes each tab has the same size" should be shot

Reply:
How about this?
![image](https://user-images.githubusercontent.com/11050425/79016559-35361f80-7b24-11ea-937e-487a6e6a8c01.png)

Reply:
You didn't change the sentence I pointed out, so, no?

Reply:
omg. I can't believe I didn't link the updated schema haha. Here's the _actual_ updated copy:
![image](https://user-images.githubusercontent.com/11050425/79020953-24d77200-7b2f-11ea-88ed-771b30124d97.png)

Reply:
(basically removed "has". It didn't sound right)

Reply:
thanks. yeah.